src/core/filters/output/BlackWhiteOptions.cpp

```diff
@@ -72,6 +72,12 @@ BinarizationMethod BlackWhiteOptions::parseBinarizationMethod(const QString& str
     return WOLF;
   } else if (str == "sauvola") {
     return SAUVOLA;
+  } else if (str == "edgeplus") {
```

Comment:
What do you think about refactoring this one to a switch/case? Just for readability?
Just like in `QString BlackWhiteOptions::formatBinarizationMethod(BinarizationMethod type) `

Reply:
Hi @vigri .

`switch/case` doesn't work on strings.

Reply:
Sorry, my fault. You are right, we are here in cpp :)